embedded/store/immustore.go

```diff
@@ -173,8 +174,9 @@ type refVLog struct {
 }
 
 func Open(path string, opts *Options) (*ImmuStore, error) {
-	if !validOptions(opts) {
-		return nil, ErrIllegalArguments
+	err := opts.Validate()
```

Comment:
originally it was catching up the case when opts was nil, now it will panic...

Reply:
It will still catch this case, `Validate` is doing this check too.